public/components/Notifications/NotificationForm.tsx

```diff
@@ -168,18 +180,26 @@ export const NotificationForm: React.FC<NotificationFormProps> = ({
                   />
                 </EuiCompressedFormRow>
               </EuiFlexItem>
-              {prepareMessage && (
-                <EuiFlexItem>
-                  <EuiCompressedFormRow>
-                    <EuiSmallButton
-                      fullWidth={false}
-                      onClick={() => prepareMessage(true /* updateMessage */)}
-                    >
-                      Generate message
-                    </EuiSmallButton>
-                  </EuiCompressedFormRow>
-                </EuiFlexItem>
-              )}
+
+              <EuiFlexItem>
+                <EuiCompressedCheckbox
+                  id={`checked`}
```

Comment:
Seems too generic. Can we add something like `notification-message-preview-checkbox`? also we can simply do `id='<ID>'`

Reply:
Done